recipes/pythermo/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "pythermo" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pythermo-{{ version }}.tar.gz
+  sha256: 2fd72f4406472d3188a274e9df870e02543868825f12f04a970a42e9e0f0efb4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - scipy
+    - matplotlib-base
+
+test:
+  imports:
+    - pythermo
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/pythermo/
+  summary: Tools for performing computational tasks in the field of low-temperature thermochronology.
+  dev_url: https://github.com/OpenThermochronology/PyThermo
+  license: GPL-3.0
```

This is the correct version according to the doc linked.
```suggestion
  license: GPL-3.0-only
```